### PR TITLE
Fix incorrect BHL link.

### PR DESCRIPTION
**Pull request by charvolant**
And make both BHL and trove links configurable as God intended.

Fix for https://github.com/AtlasOfLivingAustralia/bie-plugin/issues/176

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+dist: trust
```
Comment (ansell) [inline .travis.yml]: `dist: trusty`, minor typo

File: grails-app/views/species/_literature.gsp [hunk 2]
```diff
@@ -16,16 +16,15 @@
         <div class="col-md-9" style="padding-top:14px;">
 
             <div id="bhl-integration">
-                <h3>Name references found in the <a href="http://biodiversityheritagelibrary.com/" target="_blank">Biodiversity Heritage Library</a></h3>
+                <h3>Name references found in the <a href="${grailsApplication.config.literature.bhl.url}/search?SearchTerm=%22${synonyms?.join('%22+OR+%22')}%22&SearchCat=M#/names" target="_blank">Biodiversity Heritage Library</a></h3>
                 <div id="bhl-results-list" class="result-list">
-                    <a href='http://www.biodiversitylibrary.org/search?SearchTerm=%22${synonyms?.join('%22+OR+%22')}%22&SearchCat=M#/names' target='bhl'>Search BHL for references to ${tc?.taxonConcept?.nameString}</a>
```
Comment (ansell) [inline grails-app/views/species/_literature.gsp]: Not familiar with the code, but is this div automatically filled by a browser script?

File: .travis.yml (hunk 1 above)
Comment (charvolant) [reply .travis.yml]: Fixed

File: grails-app/views/species/_literature.gsp (hunk 2 above)
Comment (charvolant) [reply grails-app/views/species/_literature.gsp]: Yes. I've added in a comment to clarify this. The link was inconsistent with the trove behaviour below, so I've made them consistent.

**Pull request merged**